src/app/main/services/notification.service.ts

```diff
@@ -17,16 +17,22 @@ export class NotificationService {
     // action: string = 'x', duration: number = 4200
     // and / or type: 'note' | 'warning' | 'error' | 'success'; which can be used for the panelClass
     openSnackBar(notification: string | ApiResponseError, type?: 'success' | 'error'): void {
-        let duration = 5000;
+        // let duration = 5000;
```

Comment:
Why not deleting this variable if it is not used?

Reply:
I thought I deleted it already... will do it asap

Reply:
done in 94c4c301